tests/unit/s2n_pq_kem_test.c

```diff
@@ -54,6 +54,23 @@ int main()
 {
     BEGIN_TEST();
 
+#if defined(OPENSSL_IS_AWSLC) && defined(AWS_LC_API_VERSION)
+    const unsigned long lc_vers = awslc_api_version_num();
+#else
+    const unsigned long lc_vers = SSLeay();
+#endif
+
+    /* AWS-LC did not always expose an API version. Now that it does, it only occupies the
+     * lower byte of an unsigned long. OpenSSL and AWS-LC expose _OpenSSL_'s version
+     * number through SSLeay(), and OpenSSL's version occupies most bytes of the long.*/
```

Comment:
```suggestion
    /*  Check that we are comparing the API version from awslc.
     *
     * AWS-LC did not always expose an API version. Now that it does, it only occupies the
     * lower byte of an unsigned long. OpenSSL and AWS-LC both expose _OpenSSL_'s
     * version number through SSLeay(), but OpenSSL's version occupies most bytes of
     * the long (ie. 0x1010107f).
     */
```

Reply:
> is there a better name for the variable is_lc_vers_api_vers. Maybe is_awslc_api_version

sure, that works. i'll change it to `is_awslc_api_version`.

> an explicit comment that states what you are checking (added a comment above)

i think most of your re-wording works well and i'll incorporate it, but the latinism "i.e." specifies; i don't think we want to fix a version number that will surely change (i.e. `0x1010107f` in this case) . perhaps ["e.g."](https://en.wiktionary.org/wiki/e.g.) is better?

Reply:
Haha, yep, e.g. is more appropriate here and what i meant.

Reply:
i ended up ripping this &'ing + comment out.